web-client/src/pages/CartPage.tsx

```diff
@@ -1,5 +1,93 @@
-import UnderConstruction from "@/components/UnderConstruction";
+import React from 'react';
+import CartItem from '../components/CartItem';
+import { Button } from '@/components/ui/button';
+import { Card } from '@/components/ui/card';
 
-export default function CartPage() {
-  return <UnderConstruction pageName="Cart" />;
-}
+// Hard-coded items for testing purposes
+const items = [
+  {
+    sockId: 1,
+    sockVariantId: 12,
+    name: 'Socks (awesome)',
+    size: 'XL',
+    price: 21.20,
+    quantity: 2,
+    imageUrl: 'https://via.placeholder.com/150',
+  },
+  {
+    sockId: 2,
+    sockVariantId: 13,
+    name: 'Retro Socks',
+    size: 'M',
+    price: 15.50,
+    quantity: 1,
+    imageUrl: 'https://via.placeholder.com/150',
+  },
+];
+
+const CartPage: React.FC = () => {
```

Comment:
To be consistent with the rest of the code, just use a regular function export like this:

```ts
export default function CartPage() {
...
``

Reply:
Fixed